bin/init

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# This script will setup the database and then start the rails application
+
+set -e
+# check that BUNDLER_BIN is set
+if [ ! -z $BUNDLER_BIN ]; then
+    echo "ERROR: BUNDLER_BIN var is not set. Please set that and try again" >&2
+    exit -1
+fi
+
+if [ ! -z $PORT ];then
+    echo "WARNING: PORT var not set. Setting to 80" >&2
+    PORT=80
+fi
+
+setup_database() {
+  set +e
+
+  TIMEOUT=90
+  COUNT=0
+  RETRY=1
+
+  export SKIP_MIGRATION=1
+
+  while [ $RETRY -ne 0 ]; do
+    case $(SKIP_MIGRATION=1 $BUNDLER_BIN exec rails r /check_db.rb | grep DB) in
```

Comment:
We can directly call to `bundle` instead of using `$BUNDLER_BIN`. We aren't setting it and since this runs in a containerized context we know that `bundle` is on the `PATH`.